src/Arcus.Observability.Tests.Unit/Arcus.Observability.Tests.Unit.csproj

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="Bogus" Version="34.0.1" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.0.1" />
```

Comment:
Do we need this package here ?
(Maybe to get hold of a SqlConnection :? but since we're in unit-tests here, I wonder if that 's really necessary ?

Reply:
Because we need the `SqlConnectionStringBuilder` to validate our test outcomes.

Reply:
Hm, ok.  Taught we would just do some plain old asserts since we know actually what to expect I assume.  But ok.

Reply:
We do not exactly know as all is randomized inputs, so we would have to write a parser dedicated to the tests to find out. Therefor the `SqlConnectionStringBuilder` is a great option to make sure that the new functionality acts the same as the old.
Note that not only the key/value pairs are randomized, we also randomize whitespace, aliases, positions and character casing.